README.md

```diff
@@ -268,6 +268,36 @@ Variable | Description | Default value
 `DISABLE_HTTPS` | Disable HTTPS, this can be useful if TLS connections are going to be handled outside of this setup | 1
 `ENABLE_HTTP_REDIRECT` | Redirects HTTP traffic to HTTPS | 1
 
+#### Setting up Octo (cascaded bridges)
```

Comment:
This needs to be moved to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md